tests/VObject/Recur/EventIterator/HandleRDateExpandTest.php

```diff
@@ -54,7 +54,125 @@ function testExpand() {
 
         $result = array_map(function($ev) {return $ev->dtstart->getDateTime();}, $result);
         $this->assertEquals($expected, $result);
-    
+
+    }
+
+
+    function testExpandRDateWithoutTime() {
+        $input = <<<ICS
+BEGIN:VCALENDAR
+VERSION:2.0
+BEGIN:VEVENT
+UID:BDF720338B3EC8D2C09CE95CFF85C0B1-A26036CE0B219773
+DTSTAMP;VALUE=DATE-TIME:20151211T121402Z
+CREATED;VALUE=DATE-TIME:20151013T061627Z
+LAST-MODIFIED;VALUE=DATE-TIME:20151014T093742Z
+DTSTART;VALUE=DATE-TIME;TZID=Europe/Berlin:20151028T124500
+DTEND;VALUE=DATE-TIME;TZID=Europe/Berlin:20151028T141500
+SUMMARY:test summary
+DESCRIPTION:test desc
+SEQUENCE:1
+RDATE;VALUE=DATE:20151028,20151111,20151125,20151209,20160113,20160120,2016
```

Comment:
Well, if the syntax is supported by iCalendar, it would have a very specific meaning, and it's not "inherit the timestamp from the master event". The only way I could see that this should logically behave, is that subsequent events become `DATE`-only 'floating date' events.

While I agree that the spec does not appear to specifically forbid it, it also definitely does not suggest inheriting the time if only a date is specified in `RDATE`.

If Kolab is the only thing generating this, then I think that it's better to fix Kolab. I'm also curious if you can link me to the Lightning bugzilla ticket, as I somewhat suspect that Phillip will say the same thing.